### PR TITLE
Allow ChefSpec tests to be run on downstream cookbooks

### PR DESCRIPTION
**Pull request by EdHurtig**

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@
 storage = EverTools::Storage.new(node)
 ephemeral_mounts = []
 
-if File.readlines('/proc/mounts').grep(%r{/mnt/dev0}).size.zero?
+if File.exist?('/proc/mounts') && File.readlines('/proc/mounts').grep(%r{/mnt/dev0}).size.zero?
```
Comment (jeffbyrnes) [inline recipes/default.rb]: This is a nice catch.

File: libraries/storage.rb [hunk 2]
```diff
@@ -9,8 +9,12 @@ def dev_names
       elsif @node['etc']['passwd']['vagrant']
         Chef::Log.debug('Using vagrant storage')
         names = vagrant_dev_names
+      elsif defined?(ChefSpec)
+        Chef::Log.debug('Chefspec Detected, skipping mounts')
```
Comment (jeffbyrnes) [inline libraries/storage.rb]: @EdHurtig don’t forget, you may omit parentheses generally for single-line methods with no chaining.

Comment (EdHurtig) [inline libraries/storage.rb]: Oops, That's some PHP falling through there...

**Pull request merged**